### PR TITLE
Feature/start trip message build sms

### PR DESCRIPTION
**Pull request by bhohnco**
## Type of Change
- [ ] JSX
- [ ] HTML
- [ X ] JavaScript
- [ ] CSS
- [ X ] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ X ] Twilio API
- [ X ] Refactor/Fix Bugs

## Please describe the change made
Messaging through the FE and BE is working and has been tested
Start Trip Message is firing on begin trip click 
Props from the contact information and trip information are being passed into the start message function
The start Text message displays the user's name, start location, end location, and ETA. 
Trip Complete Modal is created but needs to be troubleshot/hooked up correctly. 

## Please give a short description of the changes that were made below

- Fix:

- Feature:
Messaging through the FE and BE is working and has been tested
Start Trip Message is firing on begin trip click 
Props from the contact information and trip information are being passed into the start message function
The start Text message displays the user's name, start location, end location, and ETA. 
Trip Complete Modal is created but needs to be troubleshot/hooked up correctly. 

## How has this been tested?
- [X ] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman
- [X ] Mobile

### CODE REVIEW

File: cypress/integration/mainpage.spec.js [hunk 1]
```diff
@@ -53,14 +53,28 @@ beforeEach(() => {
   it('Should be able to type into the search input and see that value in the start point input', () => {
     cy.get('.start-point').type('6245 Garrison St, Arvada, CO 80004')
         .should('have.value', '6245 Garrison St, Arvada, CO 80004')
+        .wait(2000)
```
Comment (pcmueller) [inline cypress/integration/mainpage.spec.js:56]: Nice, looks like we had been failing this test - setting a timeout is a good idea

File: cypress/integration/usereta.spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  it('Should be able to type into the search input and see that value in the end point input', () => {
+    cy.get('.end-point').type('1602 South Garfield St, Denver, CO 80206')
+        .should('have.value', '1602 South Garfield St, Denver, CO 80206')
+        .wait(2000)
+        .type("{enter}")
+
+  })
+
+  it('Should be able to select option from dropdown menu for transportation options', () => {
+    cy.get('[class*="select-transport"]').click({force: true})
```
Comment (pcmueller) [inline cypress/integration/usereta.spec.js:37]: Ooh I've never seen this class-selector syntax before - you'll have to run me through that one

**Review by pcmueller — APPROVED** [review]
Great job with the testing!  It's a relief we won't have to do so much of that in our last couple days.

**Review by foymikek — APPROVED** [review]
Testing is great and code looks clean!

**Pull request merged**